RealmSwift/Tests/SwiftTestObjects.swift

```diff
@@ -442,6 +442,18 @@ class SwiftObjectiveCTypesObject: Object {
     @objc dynamic var numCol: NSNumber? = 0
 }
 
+class SwiftComputedPropertyNotExplicitlyIgnoredObject: Object {
+    dynamic var _urlBacking = ""
+    var url: URL? {
```

Comment:
Shouldn't this be marked as `dynamic`, as all other Realm properties that would otherwise have been considered "managed" be configured? Or perhaps it'd be wise to have both defined here to confirm that we do the right thing in all cases.